.openpublishing.redirection.json

```diff
@@ -997,6 +997,14 @@
             "redirect_url": "/aspnet/mvc/overview/deployment/docker-aspnetmvc",
             "redirect_document_id": true
         },
+        {
+            "source_path": "docs/framework/docker/console.md",
```

Comment:
Instead of sending redirects to the framework guide, maybe redirect to our .NET Core Docker content: `/dotnet/core/docker/intro-net-docker`

Reply:
Yeah, I was thinking about that too. I wasn't sure if that would just be more confusing because they would think something was wrong, they wanted framework not core... It's too bad we really do not have a "Sorry this content has been removed" type of link we can send people too besides leaving a 404.